enterprise/server/test/integration/ci_runner/ci_runner_test.go

```diff
@@ -690,7 +690,7 @@ func TestHostedBazel_ApplyingAndDiscardingPatches(t *testing.T) {
 			"--target_repo_url=file://" + targetRepoPath,
 			"--target_branch=master",
 			"--cache_backend=" + app.GRPCAddress(),
-			"--patch_digest=" + fmt.Sprintf("%s/%d", patchDigest.GetHash(), patchDigest.GetSizeBytes()),
+			"--patch_digest=" + fmt.Sprintf("blobs/%s/%d", patchDigest.GetHash(), patchDigest.GetSizeBytes()),
```

Comment:
if we add the instance name here would that let us remove the backwards-compatibility code?

Reply:
The test does not set a remote_instance_name, so no changes are required here. I did change hosted_runner.go though which sets these -- now it always passes a full correct download string URI, and I removed the fallback code.